index.js

```diff
@@ -0,0 +1,10 @@
+var twitterText = require('twitter-text');
+
+module.exports = function(text, imageAttachments) {
+  var charsLeft = 140;
+
+  // Images (1 or more), use up 24 characters.
+  if (imageAttachments === true) { charsLeft -= 24; };
+
+  return charsLeft - twitterText.getTweetLength(text);
+};
```

Comment:
Kind of. You probably don't *need* a symbolic constant; a variable will do for readability / removing magic numbers.

Constants can be done though: https://www.youtube.com/watch?v=hQVTIJBZook#t=28m9s